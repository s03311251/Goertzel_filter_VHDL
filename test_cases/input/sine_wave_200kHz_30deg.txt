2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
2FFF
3F4C
2358
02C4
0A96
