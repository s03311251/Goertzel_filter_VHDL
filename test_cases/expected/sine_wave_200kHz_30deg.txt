FFFFFF80
FFFFFFE0
